metron-interface/metron-rest/src/test/java/org/apache/metron/rest/controller/SearchControllerIntegrationTest.java

```diff
@@ -132,6 +132,30 @@ public void testDefaultQuery() throws Exception {
     sensorIndexingConfigService.delete("bro");
   }
 
+  @Test
+  public void testDefaultColumnMetadata() throws Exception {
```

Comment:
Would you mind adding a comment somewhere that describes in more detail what this test case is testing?  Just looking at this test case in isolation, it is not clear to me what we are testing.

Reply:
Let me explain better.  I assume this is testing that the set of default indices are used.  That's just not clear to me how we are verifying that here.

Whereas, in your other test case `getColumnMetadataShouldProperlyGetDefaultIndices` that makes it very clear to me what you are testing. :)

Reply:
Done